CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.20.0 (June 16, 2025)
+
+NOTES:
+
+* **This release requires Juju controller version 2.9.49 or higher Juju.**
+* **If using JAAS, this release requires Juju controller version 3.6.5 or higher.**
+* This release uses Juju client api code from the Juju 3.6.4 release.
+
+ENHANCEMENTS
+
+* **BREAKING CHANGE** to the `juju_offer` schema. The `endpoint` field is field has been removed in favor of the `endpoints` field, which allows for definition of multiple endpoints in a single juju application offer [752](https://github.com/juju/terraform-provider-juju/pull/752) by @SimoneDutto.
```

Comment:
Maybe a note to a document on migrating? As I know the Terraform state will automatically migrate.